originals/w.yaml

```diff
@@ -1,3 +1,18 @@
+- name: ' Warcraft 2000'
+  names:
+  - ' Warcraft 2000: Nuclear Epidemic'
+  external:
+    website: 'https://www.myabandonware.com/game/warcraft-2000-nuclear-epidemic-bip'
```

Comment:
don't know if this is the best website - there isn't an official page for this game